modules/administration/pages/custom-channels.adoc

```diff
@@ -25,7 +25,18 @@ Many third party vendors secure packages with GPG.
 If you want to use GPG-protected packages in your custom channel, you need to trust the GPG key which has been used to sign the metadata.
 You can then check the [guimenu]``Has Signed Metadata?`` check box to match the package metadata against the trusted GPG keys.
 // We need to create a section on importing GPG keys and change this to point there: https://github.com/SUSE/spacewalk/issues/9474 LKB 2019-09-18
-For more information on importing GPG keys, see xref:reference:systems/autoinst-gpg-and-ssl-keys.adoc[].
+// This xref is misleading:
+// For more information on importing GPG keys, see xref:reference:systems/autoinst-gpg-and-ssl-keys.adoc[].
+
+If remote channels and repositories are signed with GPG keys, you can import and trust these GPG keys.
+For example, execute the [command]``spacewalk-repo-sync`` from the command line on the {productname} Server:
+
+----
+/usr/bin/spacewalk-repo-sync -c <channellabelname> -t yum
+----
+
+The underlying [command]``zypper`` call will import the key, if it is available.
+The {webui} does not offer this feature.
```

Comment:
This works only, when the repository you want to mirror is setup in a special way and provide the "key" in the repo next to the signature. This is the case for all repos generated by OBS, but not typical for other repos from other OSes.

For them you need to manually import the GPG key into the keyring.

It should be possible to import it into `/var/lib/spacewalk/gpgdir` keyring using gpg commandline tool.
This would be kind of permanent and the key would also persist if the chroot env would be cleaned.
Not sure how it is possible to import the key into the RPM DB which is used for syncing.
@meaksh is this correct and what would you do?

The gpg command to import a key into the pub keyring is:
```
$> gpg --keyring /var/lib/spacewalk/gpgdir  --import /path/to/gpg.key
$> gpg --edit-key <keyid>
```
Using gpg is horrible. You need to find out the keyid, you need to trust "ultimately" and save the change.

Reply:
I added this info.  I'd say it is ready for review.  Maybe, we can add more info/details later.